### PR TITLE
ICU-20979 BRS67RC Uploading ICU4C API docs.

### PR DESCRIPTION
**Pull request by srl295**
sibling to #10

### CODE REVIEW

File: apidoc/released/icu4c/appendable_8h_source.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
  <tbody>
  <tr style="height: 56px;">
   <td id="projectalign" style="padding-left: 0.5em;">
-   <div id="projectname">ICU 66.1
```
Comment (jefgen) [inline apidoc/released/icu4c/appendable_8h_source.html:21]: It's too bad that this is in every single file. If only HTML had a good way to #include another file, then it wouldn't need to rev every file...

**Comment by markusicu** [comment]
the docs should still go into dev/ not released/ right? so that people going to the stable/public version don't yet see the 67rc docs?

**Comment by jefgen** [comment]
nit: It looks like many of the diffs from permissions changes.
100755 → 100644
Could probably cut down on the churn if it was the same.

**Comment by jefgen** [comment]
> the docs should still go into dev/ not released/ right? so that people going to the stable/public version don't yet see the 67rc docs?

Oh good catch.

**Comment by srl295** [comment]
> nit: It looks like many of the diffs from permissions changes.

I usually try to catch that, will try again

**Comment by srl295** [comment]
doing so
--
Steven R. Loomis | @srl295 | git.io/srl295

**Comment by markusicu** [comment]
I am not sure how to effectively review the diffs. For example, basictz_8h_source.html line 49 is thousands of lines long. How is anyone supposed to spot changes? (I search for & found the newly explicit operator=().) I know this won't get fixed now, but maybe submit an improvement ticket for the tool to generate shorter HTML lines?

Rant: It's also annoying that GitHub can (usually) show rich diffs for Markdown but not for HTML. Not everything can be Markdown.

Also sad: The usual preview prefix works for files but not for folders:
https://htmlpreview.github.io/?https://raw.githubusercontent.com/srl295/icu-docs/c-67-1-rc/apidoc/released/icu4c/classicu_1_1BasicTimeZone.html

**Comment by srl295** [comment]
> I am not sure how to effectively review the diffs

I'm not sure either.

Since it is just (in this case) an unzip from a doc file, perhaps in the future the best way is to somehow not have any manual steps. Then you can just review that file X was unpacked to location Y.

**Comment by markusicu** [comment]
> I am not sure how to effectively review the diffs. For example, basictz_8h_source.html line 49 is thousands of lines long.

Looks like the ..._source files are really bad, but the real docs files are reasonably sane.

**Comment by markusicu** [comment]
Please merge... or would it be ok if I clicked the green button?

**Comment by markusicu** [comment]
Thanks! :-)

**Pull request merged**